ui/raidboss/data/03-hw/dungeon/fractal_continuum.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+// Fractal Continuum
+[{
+  zoneRegex: /^The Fractal Continuum$/,
+  triggers: [
+    {
+      id: 'Fractal Rapid Sever',
+      regex: / 14:F7A:Phantom Ray starts using Rapid Sever on (\y{Name})/,
+      infoText: function(data, matches) {
+        if (data.me == matches[1]) {
+          return {
+            en: 'Tank buster on YOU',
+          };
+        }
+        if (data.role == 'healer') {
+          return {
+            en: 'Buster on ' + data.shortName(matches[1]),
+          };
+        }
+      },
+    },
+    {
+      id: 'Fractal Slash',
+      regex: / 14:F83:Minotaur starts using 10-Tonze Slash/,
+      infoText: {
+        en: 'Out of front',
+      },
+    },
+    {
+      id: 'Fractal Swipe',
+      regex: / 14:F81:Minotaur starts using 11-Tonze Swipe/,
+      infoText: {
+        en: 'Out of front',
+      },
+    },
+    {
+      id: 'Fractal Small Swing',
+      regex: / 14:F82:Minotaur starts using 111-Tonze Swing/,
+      infoText: {
+        en: 'Get out',
+      },
+    },
+    {
+      id: 'Fractal Big Swing',
+      regex: / 14:F87:Minotaur starts using 1111-Tonze Swing/,
+      alertText: {
+        en: 'Use a cage',
+      },
+    },
+    {
+      id: 'Fractal Aetherochemical Bomb',
+      regex: / 1A:(\y{ObjectId}):(\y{Name}) gains the effect of Aetherochemical Bomb/,
+      condition: function(data) {
+        return data.CanCleanse;
```

Comment:
Sorry, one more thing: this is a function, and not just a property.

Reply:
Oops, so it is.